### DISCUSSION TITLE
Threat Prevention/IPS

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#7581] [Unsupported Versions] [answered]
Hey,

Is it possible to use Security Onion as Intrusion Prevention System (IPS)?

**Comment by 0nnyx** [comment]
As far as I know, Security Onion never proposed IPS features as it's passively listening to traffic on a Lan segment. You might rather look at Pfsense if you want an IPS solution

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/faq.html#can-security-onion-run-in-ips-mode:
> [](https://docs.securityonion.net/en/2.3/faq.html#can-security-onion-run-in-ips-mode)Can Security Onion run in IPS mode?
We do not support IPS.

**Reply by corycandia** [reply]
Just curious why SO isn't moving towards IPS?  Or are you considering it already?  Too many alternatives in the space?

Analysis is great, but isn't breaking an attack chain better than alerting to it, or finding it after the fact?

Not criticism, love SO.

**Reply by dougburks** [reply]
Most organizations have some sort of Next Generation Firewall (NGFW) with IPS features and that is the proper place for blocking to occur. Security Onion is designed to monitor the traffic that makes it through your firewall.

I've added this to https://docs.securityonion.net/en/2.3/faq.html#can-security-onion-run-in-ips-mode.
